specification/compute/resource-manager/Microsoft.Compute/stable/2021-10-01/gallery.json

```diff
@@ -2098,6 +2098,16 @@
         "manageActions": {
           "$ref": "#/definitions/UserArtifactManage"
         },
+        "settings": {
```

Comment:
please place new properties at the end of this properties list. so after 'enabledHealthCheck" 
and please create a separate PR for changing existing version, and use this one only for 2022-03-01 change

Reply:
removed from this PR. will create a new one to update gallery.json 2021-10-1